src/python/pants/backend/awslambda/python/target_types.py

```diff
@@ -246,6 +246,11 @@ class PythonAwsLambdaRuntime(StringField):
         """
         The identifier of the AWS Lambda runtime to target (pythonX.Y).
         See https://docs.aws.amazon.com/lambda/latest/dg/lambda-python.html.
+
+        In general you'll want to define either a `runtime` or one `complete_platforms` but not
```

Comment:
What happens today if you specify both? I assume we'd want to error? And wouldn't that error message be sufficient??

Reply:
Currently, like any other PEX, multiplatform is supported and all platforms and complete_platforms are additive. So you can make a PEX that can be deployed to >1 lambda runtime. A separate step could be the whole deprecation procedure to restrict this down to one, which would include transitioning to platform XOR complete_platforms and making complete_platforms singular.